Microsoft.Toolkit.Uwp.SampleApp/Common/XAMLHelper.cs

```diff
@@ -0,0 +1,70 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Reflection;
+using Microsoft.Toolkit.Uwp.UI.Animations;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp
+{
+    public static class XAMLHelper
```

Comment:
Should this be `AcrylicHelper`?  Would help in readability when used.

Reply:
I had it named AcrylicHelper originaly, but I wanted to also add helpers for Reveal later on so renamed it to be more generic

Reply:
Name should be XamlHelper